armi/nuclearDataIO/cccc/cccc.py

```diff
@@ -372,6 +243,11 @@ def _rwMatrix(contents, func, *shape):
         This can be important for performance when reading large matrices (e.g. scatter
         matrices). It may be worth investigating ``numpy.frombuffer`` on read and
         something similar on write.
+
+        It seems that with shape, the first shape argument should be the outermost loop.
+
+        So if you have ``((MR(I,J),I=1,NCINTI),J=1,NCINTJ)`` you would pass in
+        the shape as (NCINTJ, NCINTI).
```

Comment:
I think you can flip numpy.ndarray to use column major. No biggie though

Reply:
you can use `order="F"` to change how they're laid out in memory but the indexing stays constant I believe. But you're probably right that there's a better way to do this using column major fortran order. I'll at least babble about it in comments.